fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,292 @@
+	to_chat(A, "<span class='danger'>You swiftly and repeatedly slash at [D], truly a master attack!</span>")
+	A.adjustStaminaLoss(80)
+	var/obj/item/bodypart/limb = D.get_bodypart(ran_zone(A.zone_selected)) /// Guaranteed, unlike Flying Axe Kick
+	var/datum/wound/slash/moderate/crit_wound = new
+	crit_wound.apply_wound(limb)
+	D.apply_damage(30, BRUTE)
+	playsound(get_turf(A), 'sound/weapons/bladeslice.ogg', 50, TRUE, -1)
+	return TRUE
+
+
+/// Intents
+/datum/martial_art/velvetfu/harm_act(mob/living/A, mob/living/D)
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("H",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "harmed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = "silken wrist"
+	var/bonus_damage = 10
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 5
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// No grabbing in Velvet-Fu!
+/datum/martial_art/velvetfu/grab_act(mob/living/A, mob/living/D)
+	if(HAS_TRAIT(A, TRAIT_PACIFISM))
+		return FALSE
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("G",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "grabbed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = pick("ascending claw", "descending claw")
+	var/bonus_damage = 10
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 5
+		picked_hit_type = "iron hooved"
+	D.apply_damage(bonus_damage, A.get_attack_type())
+	D.visible_message("<span class='danger'>[A] [picked_hit_type]ed [D]!</span>", \
+					"<span class='userdanger'>You're [picked_hit_type]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [picked_hit_type] [D]!</span>")
+	playsound(D, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// No shoving in Velvet-Fu!
+/datum/martial_art/velvetfu/disarm_act(mob/living/A, mob/living/D)
+	if(HAS_TRAIT(A, TRAIT_PACIFISM))
+		return FALSE
+	var/datum/dna/dna = A.has_dna()
+	if(dna?.check_mutation(HULK))
+		return FALSE
+	add_to_streak("D",D)
+	if(check_streak(A,D))
+		return TRUE
+	log_combat(A, D, "disarmed (Velvet-Fu)")
+	A.do_attack_animation(D)
+	var/picked_hit_type = "side kick"
+	var/bonus_damage = 10 // Kicking deals more damage
+	if(D.body_position == LYING_DOWN)
+		bonus_damage += 8
```

Comment:
mfw the disarm replacement for velvet fu is mechanically just a copy+paste of the harm move, but with +3 damage to prone targets

I guess the combos distinguish them? most of the combos aren't worth it at all, though.

Reply:
Its meant to be Lisa's combo dial